kitsune/users/views.py

```diff
@@ -525,9 +527,13 @@ def edit_profile(request, username=None, template=None):
 
     # TODO: detect timezone automatically from client side, see
     # http://rocketscience.itteco.org/2010/03/13/automatic-users-timezone-determination-with-javascript-and-django-timezones/  # noqa
+    msgs = messages.get_messages(request)
```

Comment:
Can't we just add a message in the `auth.py` like here: 
https://github.com/mozilla/kitsune/blob/firefox-accounts-integration/kitsune/users/auth.py#L127

Reply:
@akatsoulas I did that here: https://github.com/mozilla/kitsune/pull/3588/files#diff-ce68f8e5c4103e3bd04a6d97db5eef2dR129

The purpose of the list comp is to filter out non-FXA related messages.  I could just do that in the template if you'd prefer